### ISSUE TITLE
Started only playing one song from the queue.

### ISSUE THREAD

**Issue by toxicated101** [Tyzer34/plexMusicPlayer#29]
For about two weeks Alexa has only been playing one song from the queue. If I tell Alexa "Next" it will play the next song and then stop again. I added some logging and it appears to get to:-

@ask.on_playback_finished()
def play_back_finished(token):

and select the correct song and then go_next is called but no audio play. This was working so I guess it's something that has changed on Alexa. 

[plexMusicPlayer.txt](https://github.com/Tyzer34/plexMusicPlayer/files/1056299/plexMusicPlayer.txt)

Log with flask-ask logging turned on
[Log2.txt](https://github.com/Tyzer34/plexMusicPlayer/files/1056428/Log2.txt)

**Comment by WaferMouse** [comment]
It looks as though you're having the same issue that I'm having with my fork downstream. From your log it looks as though when it hangs it forces you to skip a track, the same as me.

I'm no closer to a fix than you are, but thanks for confirming that it used to work correctly.

**Comment by toxicated101** [comment]
I've tried to go back to basics and tested the flask-ask playlist sample.  I manage to play about 3 songs before it does the same thing, even though there is a next track which works when I try the url directly it stops, so I think it's an issue with flask-ask. If I can get my head around node.js and alexa-app audio implementation I might try to see if I that works. 

Nice to know I'm not the only one experiencing the issue.

**Comment by WaferMouse** [comment]
Yeah, I just saw the issue you've raised over at flask-ask.  Right now I'm trying to figure out how I can view request values in Apache so I can start trying to debug.

**Comment by redspike** [comment]
Just to say I have this issue as well, though I manage to play two tracks before having to ask for "next"...

**Comment by WaferMouse** [comment]
@redspike yeah, it seems inconsistent for me, sometimes 1 track, sometimes 2.

Flask-ask has had a breaking commit since Plexmusic was written, and at least one commit relating to audio, so now that I have Apache outputting some debug I plan to step through the updates and see if I can't fix this issue along the way.

**Comment by redspike** [comment]
Is it just me or is it working properly again? I've had it playing at least 6 songs in a row now...

**Comment by toxicated101** [comment]
It's not working for me yet, still only plays one track at a time.

**Comment by redspike** [comment]
Have you updated to Flask-ask 0.9.4? I'm trying mine again tonight and it's working perfectly again. I will say that I upgraded to 0.9.4 and still had that issue 2 weeks ago, it's magically disappeared now...

**Comment by toxicated101** [comment]
Your right , upgraded flask and it does appear to be working, 3 songs in so far. lets hope it stays working :)
